### PR TITLE
Update links in footer

### LINKED ISSUE #728: Footer: add links to images, remove FB and Instagram, fix Samvera link
### Descriptive summary

- Remove Instagram logo (no account)
- Remove Facebook logo (no account)

- Add links to the following:
  - OSU logo - https://oregonstate.edu
  - UO logo - https://www.uoregon.edu
  - Twitter logo - https://twitter.com/oregondigcol

- Make text link after Twitter logo to '@oregondigcol'

- Fix Samvera link (no https)

### Expected behavior

All logos and graphics have active and correct links.

### Related work

#597 

### Accessibility Concerns

Add any information here to indicate any known or suspected accessibility issues for this ticket

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #728

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -29,14 +28,13 @@
...
-            <img class='social-logo' src='<%= image_url('facebook.png')%>' alt="Facebook">
-            <img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter">
+            <a href="https://twitter.com/oregondigcol"><img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter"></a>
+            <a class="footer-link-text" href="https://twitter.com/oregondigcol">@oregondigcol</a>
           </p>
           <p class='deployed-version'>
             Oregon Digital <%= "#{OregonDigital::VERSION}-#{ENV.fetch('DEPLOYED_VERSION', 'development')}" %>
           </p>
-          <p>Powered by <a class="footer-link-text" href="samvera.org">Samvera</a> Hyrax <%= Hyrax::VERSION %></p>
+          <p>Powered by <a class="footer-link-text" href="http://www.samvera.org">Samvera</a> Hyrax <%= Hyrax::VERSION %></p>
```
Comment (CGillen) [inline app/views/shared/_footer.html.erb]: https all the things! 😄

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -29,14 +28,13 @@
       <div class="col-md-4 col-xs-5">
         <div class="text-right">
           <p>
-            <img class='social-logo' src='<%= image_url('insta.png')%>' alt="Instagram">
-            <img class='social-logo' src='<%= image_url('facebook.png')%>' alt="Facebook">
-            <img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter">
+            <a href="https://twitter.com/oregondigcol"><img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter"></a>
+            <a class="footer-link-text" href="https://twitter.com/oregondigcol">@oregondigcol</a>
```
Comment (CGillen) [inline app/views/shared/_footer.html.erb]: If the image and text are in the same a-tag then you can blank the alt text on the image (alt="") and it makes tabbing easier

File: app/views/shared/_footer.html.erb (hunk 1 above)
Comment (straleyb) [reply app/views/shared/_footer.html.erb]: Ryan wick said no https here `Fix Samvera link (no https)`

Comment (CGillen) [reply app/views/shared/_footer.html.erb]: I read that as that's what's wrong with the link
`http://www.samvera.org` redirects to `https://www.samvera.org` anyways, it's just nicer for everyone involved.

Comment (straleyb) [reply app/views/shared/_footer.html.erb]: Heh. Yeah. I think i just interpreted it oddly. I fixed it.

**Pull request merged**